index.js

```diff
@@ -13,11 +13,14 @@ module.exports = class NodePolyfillPlugin {
 	}
 
 	apply(compiler) {
-		compiler.options.plugins.push(new ProvidePlugin(excludeObjectKeys({
-			Buffer: [require.resolve("buffer/"), "Buffer"],
-			console: require.resolve("console-browserify"),
-			process: require.resolve("process/browser")
-		}, this.options.excludeAliases)))
+		compiler.options.plugins.push(new ProvidePlugin({
+			...excludeObjectKeys({
+				Buffer: [require.resolve("buffer/"), "Buffer"],
+				console: require.resolve("console-browserify"),
+				process: require.resolve("process/browser")
+			}, this.options.excludeAliases),
+			...compiler.options.plugins
```

Comment:
How does this work?

Reply:
Sorry, I can't reproduce the problem. Originally, without `...compiler.options.plugins`, my React app shows an error of adm-zip using undefined process variable. After using `...compiler.options.plugins`, the problem is fixed. But now, I delete `...compiler.options.plugins` and node_modules, and run my app again, then adm-zip doesn't show error anymore.

Additionally, I use react-app-rewired + customize-cra with node-polyfill-webpack-plugin.
This is where I use node-polyfill-webpack-plugin:
https://github.com/MrMYHuang/tfwc/blob/master/config-overrides.js

I will close this PR unless I can reproduce the same problem.